src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -161,5 +161,10 @@
         <service id="Sylius\Bundle\ApiBundle\Validator\ResourceInputDataPropertiesValidatorInterface" class="Sylius\Bundle\ApiBundle\Validator\ResourceApiInputDataPropertiesValidator">
             <argument type="service" id="validator" />
         </service>
+
+        <service id="Sylius\Bundle\ApiBundle\Controller\GetStatisticsAction" public="true">
```

Comment:
Please definition to `src/Sylius/Bundle/ApiBundle/Resources/config/services/controllers.xml`